src/videojs-http-streaming.js

```diff
@@ -146,9 +146,17 @@ const emeOptions = (keySystemOptions, videoPlaylist, audioPlaylist) => {
   for (let keySystem in keySystemOptions) {
     keySystemContentTypes[keySystem] = {
       audioContentType: `audio/mp4; codecs="${audioPlaylist.attributes.CODECS}"`,
-      videoContentType: `video/mp4; codecs="${videoPlaylist.attributes.CODECS}"`
+      videoContentType: `video/mp4; codecs="${videoPlaylist.attributes.CODECS}"`,
+      pssh: ''
```

Comment:
Did we still want to default to an empty string for pssh now that we're overwriting the source options?